src/i18n/locales/zh-cn/index.js

```diff
@@ -40,7 +40,14 @@ module.exports = {
   'preview-title': '预览',
   'create': '创建 Issue',
 
+  // footer
+  'built-with': '以<a target=_blank href="https://github.com/vuejs/vue-cli">vue-cli</a>建立',
```

Comment:
Yeah indeed. Tried the markdown but it didn't work straight away, so opted for this ugly workaround. Will push an update.